sos/report/plugins/landscape.py

```diff
@@ -19,21 +19,25 @@ class Landscape(Plugin, UbuntuPlugin):
     packages = ('landscape-client', 'landscape-server')
 
     def setup(self):
-        self.add_copy_spec("/etc/landscape/client.conf")
-        self.add_copy_spec("/etc/default/landscape-client")
-        self.add_copy_spec("/etc/landscape/service.conf")
-        self.add_copy_spec("/etc/landscape/service.conf.old")
-        self.add_copy_spec("/etc/default/landscape-server")
-        self.add_copy_spec("/var/lib/landscape/landscape-oops/*/OOPS-*")
+        self.add_copy_spec([
+            "/etc/default/landscape-client",
+            "/etc/default/landscape-server",
+            "/etc/landscape/client.conf",
+            "/etc/landscape/service.conf",
+            "/etc/landscape/service.conf.old",
```

Comment:
What else is in this directory, and should we copy those as well? Or convert this over to something like `/etc/landscape/*.conf.*`?

Reply:
We need to be careful here, cause as you noticed there is a quite a few `self.do_file_sub` against files found in /etc/landscape/.

Whlie I understand why you ask the question ... I would prefer to keep this isolated per filename so that we don't miss any sensible informations sanitization by capturing too widely.

Reply:
(e.g. if customer do backup in this directory /etc/landscape/service.conf.bckp)

By copying /etc/landscape we won't be able to provide a good substitution (with regard to sensible informations) for files that doesn't match our expectation.

Reply:
Ack, I was just curious if we should extend the collection since we're already reviewing the plugin for an update.